lbrynet/extras/daemon/storage.py

```diff
@@ -892,3 +888,11 @@ def _format_claim_response(outpoint, claim_id, name, amount, height, serialized,
         "channel_name": None
     }
     return r
+
+
+def _batched_select(transaction, query, parameters):
+    for start_index in range(0, len(parameters), 900):
+        current_batch = parameters[start_index:start_index+900]
+        bind = "({})".format(','.join('?' for _ in range(len(current_batch))))
```

Comment:
What about `'?' for _ in range(len(current_batch))` -> `['?'] * len(current_batch)` ?

Reply:
+1
added that